modules/Microsoft.Web/staticSites/.test/min.parameters.json

```diff
@@ -1,9 +1,9 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "name": {
-            "value": "<<namePrefix>>-az-wss-min-001"
-        }
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
```

Comment:
All other json files have configured tab size of 4 instead of 2. Is this change intended?

Reply:
Must be from the original PR and is technically irrelevant. Will update it manually, but would suggest to ignore the tab size in the future.

Reply:
updated

Reply:
Thanks for the update. I'm not sure I agree on this one. I'd still aim for a consistent formatting througout the repo. Instead of ignoring we could probably avoid it with repo format settings.

Reply:
If we can control it with a .vscode setting, be my guest. My main concern is usually the correct formatting (e.g. that a bracket is not indented incorrectly or so) instead of the number of indents the entire file follows. E.g. the deploymentExamples don't care at all and will all auto-format the content in the readmes based on its own rules. 

So, if we can control this via a settings, great. If not - I'm not sure if it's worth the fuzz and effort reviewing ^^

Reply:
PR #1759 should do

Reply:
I think we should introduce a linting step as gate to PRs that ensure correct indentation, then the original indentations from the PRs shouldn’t be relevant.

Reply:
It appears the parameter file is aligned now. @eriqua, please resolve